pkg/pxc/backup/storage/storage.go

```diff
@@ -227,7 +227,10 @@ func (a *Azure) GetObject(ctx context.Context, name string) (io.ReadCloser, erro
 
 func (a *Azure) PutObject(ctx context.Context, name string, data io.Reader, _ int64) error {
 	objPath := path.Join(a.prefix, name)
-	_, err := a.client.UploadStream(ctx, a.container, objPath, data, nil)
+	uploadOption := azblob.UploadStreamOptions{
+		Concurrency: 4,
+	}
```

Comment:
@dcaputo-harmoni Hi! Thank you for your contribution. 
Is it possible to make this value configurable? I believe it will super useful for all our users.

Reply:
It's possible, just don't have the bandwidth to do it at the moment. Given that it's currently broken for large files what I would suggest is that we release as is to fix the issue and then just open a feature request / issue to track it and I'll get that in down the line.

Reply:
I see, time is always the problem. 
We can take over this PR and make value configurable.